projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -40,3 +40,20 @@ export function verifyTicketDetailsPageVisit(){
   cy.url().should('match',/http:\/\/.+\/my\-account\/support\-ticket\/[0-9]+/);
   cy.get('cx-customer-ticketing-messages').should('exist');
 }
+
+export function visitTicketDetailsPageForFirstTicket(){
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).should('exist');
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).then( ($ticketIdElement) => {
+    const ticketId = $ticketIdElement.text().trim();
+    cy.visit(`/my-account/support-ticket/${ticketId}`);
+    cy.get('cx-customer-ticketing-messages', { timeout: 10000 }).should('be.visible');
+    cy.url().should('include',`/my-account/support-ticket/${ticketId}`);
+  });
+}
+
+export function visitTicketDetailsPageForNonExistingTicket(){
+  cy.visit('/my-account/support-ticket/XYZ01234');
+  cy.get('cx-global-message .alert-danger', { timeout: 10000 }).should('be.visible');
+  cy.get('cx-global-message .alert-danger span').should('include.text', 'Ticket not found.');
+  cy.url().should('include','/my-account/support-ticket');
+}
```

Comment:
Maybe you can split "visitTicketDetailsPageForNonExistingTicket" function into two.
On the one hand, you can create a generic method like:
```
export function visitTicketDetailsPageForTicketId(ticketId: string) {
  const ticketDetailsPageUrl = `/my-account/support-ticket/${ticketId}`;
  cy.intercept(ticketDetailsPageUrl).as('ticketDetails');
  cy.visit(ticketDetailsPageUrl);
  cy.wait('@ticketDetails');
}
```

Like this we can have:
const NOT_EXISTENT_ID = 'XYZ01234';
customerTicketing.visitTicketDetailsPageForTicketId(NOT_EXISTENT_ID);

The second will check the existence of a banner on a page:
```
export function verifyErrorMessageIsShownWithText(errorMessage: string) {
  const TEN_SECONDS = 10000; // But Why ten seconds ?? Isn't that too much?
  cy.get('cx-global-message .alert-danger', { timeout: TEN_SECOND }).should('be.visible');
  cy.get('cx-global-message .alert-danger span').should('include.text', errorMessage);
}
```

Reply:
Moreover, there is now a function that does this:
https://github.com/SAP/spartacus/blob/f3761fa13e9541123d98e0bb1a46e6e44bdd5b8b/projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts#L60

So you will need to do:
```
export function visitTicketDetailsPageForTicketId(ticketId: string) {
  const ticketDetailsPageUrl = `/my-account/support-ticket/${ticketId}`;
  visitPage(ticketDetailsPageUrl);
}
 ```